Forms/New Student Form/New Student Form.html

```diff
@@ -334,30 +334,33 @@ <h2>General Information</h2>
 
             <!-- Third Row Questions -->
             <label for="cellPhone" class="label">Cell Phone:</label>
-            <input type="tel" pattern="([0-9]{3}) [0-9]{3}-[0-9]{4}" id="cellPhone" class="input">
+            <input type="tel" id="cellPhone" class="input" maxlength="13">
             <label for="email" class="label">E-mail Address:</label>
             <input type="email" id="email" class="input">
 
             <!-- Fourth Row Questions -->
             <label for="birthday" class="label">Birthday:</label>
-            <input type="date" id="birthday" class="input">
+            <input type="text" id="birthday" class="input" maxlength="10">
             <label for="genders" class="label">Gender:</label>
             <input list="genders" id="gender" class="input">
             <datalist id="genders" class="input">
                 <option value="Female" />
                 <option value="Male" />
-            </datalist><br><br>
+            </datalist><br>
 
             <!-- Fifth Row Questions -->
-            <div class = "inline">
-                <label for="sibling1" class="label">Sibling:</label>
+            <div>
```

Comment:
All of this is for the add and remove button to be above with the "siblings" heading